src/Makefile

```diff
@@ -572,17 +579,20 @@ help:
 	@echo ""
 	@echo "Simple examples. If you don't know what to do, you likely want to run: "
 	@echo ""
-	@echo "make -j build ARCH=x86-64    (This is for 64-bit systems)"
```

Comment:
The '-j' switch is important. Without it make will stupidly compile each .cpp file one after the other, instead of running them in parrallel.

Reply:
Corrected, thanks!